lib/Model.php

```diff
@@ -3,6 +3,7 @@
  * @package ActiveRecord
  */
 namespace ActiveRecord;
+use Closure;
```

Comment:
ah, it's only in the doc-block
you didn't put it in the actual method parameters